meetings/2024-05-29.md

```diff
@@ -0,0 +1,190 @@
+* Maxime Lecoq-Gaillard
+* [Pierre-Antoine Champin](https://champin.net/#pa)
+
+## Announcements
+
+### Meeting Guidelines
+* [W3C Solid Community Group Calendar](https://www.w3.org/groups/cg/solid/calendar).
+* [W3C Solid Community Group Meeting Guidelines](https://github.com/w3c-cg/solid/blob/main/meetings/README.md).
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Join queue to talk.
+* Topics can be proposed at the bottom of the agenda to be discussed as time allows. Make it known if a topic is urgent or cannot be postponed.
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/).
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* Virginia Balseiro
+* elf Pavlik
+
+### Introductions
+
+* Ali (via Jitsi chat):
+> Hi all,
+> 
+> I am Ali Siragedien
+> From Warsaw Poland
+> 
+> I met some of you during the Solid Symposium in Leuven
+> 
+> I canot talk now
+> 
+> I will be just listening 
+
+---
+
+## Topics
+
+### Solid CG meeting at TPAC
+
+* VB: We need to request one (or more) timeslots before June 10th. These are the options (times are expressed in local time to TPAC, which this year will be Anaheim, CA, USA, so UTC-7 hours if I am correct):
+
+Monday 23 September: 09:00 - 10:30 (16:00 - 17:30 UTC)
+Monday 23 September: 11:00 - 12:30 (18:00 - 19:30 UTC)
+Monday 23 September: 14:00 - 16:00 (21:00 - 23:00 UTC)
+Monday 23 September: 16:30 - 18:00 (23:30 - 01:00 UTC on 24 September)
+Tuesday 24 September: 09:00 - 10:30 (16:00 - 17:30 UTC)
+Tuesday 24 September: 11:00 - 12:30 (18:00 - 19:30 UTC)
+Tuesday 24 September: 14:00 - 16:00 (21:00 - 23:00 UTC)
+Tuesday 24 September: 16:30 - 18:00 (23:30 - 01:00 UTC on 25 September)
+Thursday 26 September: 09:00 - 10:30 (16:00 - 17:30 UTC)
+Thursday 26 September: 11:00 - 12:30 (18:00 - 19:30 UTC)
+Thursday 26 September: 14:00 - 16:00 (21:00 - 23:00 UTC)
+Thursday 26 September: 16:30 - 18:00 (23:30 - 01:00 UTC on 27 September)
+Friday 27 September: 09:00 - 10:30 (16:00 - 17:30 UTC)
+Friday 27 September: 11:00 - 12:30 (18:00 - 19:30 UTC)
+Friday 27 September: 14:00 - 16:00 (21:00 - 23:00 UTC)
+Friday 27 September: 16:30 - 18:00 (23:30 - 01:00 UTC on 28 September)
+
```

Comment:
~~~suggestion
* VB: We need to request one (or more) timeslots before June 10th. These are the options (times are expressed in local time to TPAC, which this year will be Anaheim, CA, USA, so UTC-7 hours if I am correct):
  ```
  Monday   23 September: 09:00 - 10:30 (16:00 - 17:30 UTC)
                         11:00 - 12:30 (18:00 - 19:30 UTC)
                         14:00 - 16:00 (21:00 - 23:00 UTC)
                         16:30 - 18:00 (23:30 - 01:00 UTC on 24 September)
  Tuesday  24 September: 09:00 - 10:30 (16:00 - 17:30 UTC)
                         11:00 - 12:30 (18:00 - 19:30 UTC)
                         14:00 - 16:00 (21:00 - 23:00 UTC)
                         16:30 - 18:00 (23:30 - 01:00 UTC on 25 September)
  Thursday 26 September: 09:00 - 10:30 (16:00 - 17:30 UTC)
                         11:00 - 12:30 (18:00 - 19:30 UTC)
                         14:00 - 16:00 (21:00 - 23:00 UTC)
                         16:30 - 18:00 (23:30 - 01:00 UTC on 27 September)
  Friday   27 September: 09:00 - 10:30 (16:00 - 17:30 UTC)
                         11:00 - 12:30 (18:00 - 19:30 UTC)
                         14:00 - 16:00 (21:00 - 23:00 UTC)
                         16:30 - 18:00 (23:30 - 01:00 UTC on 28 September)
  ```
~~~